server/portal/apps/signals/receivers.py

```diff
@@ -46,7 +46,7 @@ def portal_event_callback(sender, **kwargs):
 @receiver(post_save, sender=Notification, dispatch_uid='notification_msg')
 def send_notification_ws(sender, instance, created, **kwargs):
     # Only send WS message if it's a new notification not if we're updating.
-    logger.debug("Received a Notification event")
```

Comment:
Going off of python's [guide](https://docs.python.org/3/howto/logging.html), I think info is more appropriate than debug here, since this is just a text log statement, rather than output of a stacktrace or object.

Reply:
sounds good. we chatted offline.